### PR TITLE
#495 Django4 support

### PR DESCRIPTION
**Pull request by auvipy**
change url to re_path
use gettext instead of ugettext
if instead of ifequal in templates
no signal args

### CODE REVIEW

File: runtests/settings.py [hunk 1]
```diff
@@ -101,6 +101,8 @@
 
 ROOT_URLCONF = 'urls'
 
+DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
```
Comment (auvipy) [inline runtests/settings.py:104]: should we use default bigAutoField or add per app https://github.com/celery/django-celery-results/blob/master/django_celery_results/apps.py#L15 default_auto_field = 'django.db.models.AutoField' to keep backward compat?

File: tox.ini [hunk 2]
```diff
@@ -13,6 +14,7 @@ deps =
     django22: Django>=2.2,<3.0
     django30: Django>=3.0,<3.1
     django31: Django>=3.1,<3.2
+    django40: Django>=4.0,<4.1
     mysql: mysqlclient>=1.4.2,<1.5
```
Comment (auvipy) [inline tox.ini:18]: is there any new releases of mysqlclient that we could bump??

Comment (justquick) [reply tox.ini:18]: probably. is it breaking anything? as long as it's working for the tests i dont think it's our responsibility to test db client regressions as long as there is nothing broken

File: runtests/settings.py (hunk 1 above)
Comment (justquick) [reply runtests/settings.py:104]: that's in runtests. do you mean adding the option to `actstream.apps.ActstreamConfig` like celery did?

Comment (auvipy) [reply runtests/settings.py:104]: yes that is what I meant

File: tox.ini (hunk 2 above)
Comment (auvipy) [reply tox.ini:18]: we could try https://pypi.org/project/mysqlclient/2.1.0/ & see if it pass

**Pull request merged**